modules/http-api/src/main/java/org/mule/runtime/http/api/exception/ConnectionAlreadyClosedException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.http.api.exception;
+
+import java.io.IOException;
+
+/**
+ * This Exception is thrown when there is an error that implies losing the connection with the client.
+ * 
+ * @since 4.4.0
+ */
+public class ConnectionAlreadyClosedException extends IOException {
```

Comment:
per the comment on https://github.com/mulesoft/mule-http-connector/pull/432#discussion_r394339065, this should be general and not just part of the http api

Reply:
Do you mean that this class should be in mule-api?